rules/windows/credential_access_symbolic_link_to_shadow_copy_created.toml

```diff
@@ -15,59 +15,64 @@ index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Symbolic Link to Shadow Copy Created"
-note = """## Triage and analysis.
+note = """## Triage and analysis
 
-### Investigating
+### Investigating Symbolic Link to Shadow Copy Created
 
-Shadow copies are backups or snapshots of an endpoints files or volumes at the time of being in use. Adversaries may attempt to discover and create symbolic links to these shadow copies in order to copy sensitive information offline. If Active Directory (AD) is in use, often the ntds.dit file is a target as it contains password hashes but an offline copy is needed to extract these hashes and potentially conduct lateral movement.
+Shadow copies are backups or snapshots of an endpoints files or volumes while they are in use. Adversaries may attempt
+to discover and create symbolic links to these shadow copies in order to copy sensitive information offline. If Active
+Directory (AD) is in use, often the ntds.dit file is a target as it contains password hashes but an offline copy is
+needed to extract these hashes and potentially conduct lateral movement.
 
-#### Possible investigation steps:
+#### Possible investigation steps
 
 - Determine if a volume shadow copy was recently created on this endpoint.
 - Review priviledges of the end user as this requires administrative access.
-- Verify ntds.dit file was successfully copied and the location.
+- Verify ntds.dit file was successfully copied and its copy destination.
```

Comment:
```suggestion
- Verify the ntds.dit file was successfully copied and determine its copy destination.
```

Reply:
Added an `if` on this one, lemme know if I should change that